src/main/cli/cli.c

```diff
@@ -1276,74 +1282,109 @@ static void cliSerialPassthrough(char *cmdline)
                 pinioDtr = atoi(tok);
 #endif /* USE_PINIO */
             }
-
```

Comment:
Your examples are padding `0` for the fourth argument - this will work as it is now, but it will also be prone to breaking if users forget it, as the argument will eat up the next argument without complaining.
I'd change the examples to use `none` or so for this argument, to make it clearer that this isn't a numerical argument.
At some point we should consider doing exhaustive checking for allowable values for the arguments for this command, and throwing an error if they don't match what we expect.

Reply:
`serialpassthrough 0 115200 rxtx none 4 115200 rx`

Use none instead of 0 will make the command clearly. But if user forgot the fourth argument, change it to none is not enough, I will add some check for the new three arguments(right side port, right side baud and the mode).